### ISSUE TITLE
Question: Memory usage in webviews

### ISSUE THREAD

**Issue by teejayhh** [gokatz/ember-device#11]
I am wondering if there is any way to monitor the memory usage of a webview (ios or android/chrome) over time. Our team has a product which uses an appshell to load our canvas games which sometimes seem to leak memory and crashing the webview. It would be great if there is a way to monitor the used memory but I am not sure this scenario is covered by your addon. Do you think this is possible ?
**Update:** Maybe this answers it -> https://github.com/gokatz/ember-device/blob/master/addon/lib/memory.js#L19  totalHeap vs usedHeap Memory. Hmmm  we will give this a go and report back.

**Comment by gokatz** [comment]
Yeah. I hope those memory stats will work for your case! Waiting to see the outcomes :)
